packages/node_modules/@ciscospark/redux-module-activity/src/actions.test.js

```diff
@@ -0,0 +1,214 @@
+/* eslint-disable max-nested-callbacks */
+import {OrderedMap} from 'immutable';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+// This has to be before importing the actions so that
+// they will receive the mocked uuid
+jest.mock(`uuid`, () => (
+  {
+    v4: jest.fn(() => `jest-uuid-ftw`)
+  }
+));
+
+
+import * as actions from './actions';
+import {initialState} from './reducer';
+
+const DATE_TO_USE = new Date(`2016`);
+const _Date = Date;
+global.Date = jest.fn(() => DATE_TO_USE);
+global.Date.UTC = _Date.UTC;
+global.Date.parse = _Date.parse;
+global.Date.now = _Date.now;
+
+window.URL.createObjectURL = jest.fn();
+window.URL.revokeObjectURL = jest.fn();
```

Comment:
Should these mocks and global variables be set in the top level jest configs so that they're available everywhere?